metering/metering-uninstall.adoc

```diff
@@ -9,21 +9,17 @@ You can remove metering from your {product-title} cluster.
 
 [NOTE]
 ====
-Metering does not manage or delete Amazon S3 bucket data. After uninstalling metering,
-you must manually clean up S3 buckets that were used to store metering data.
+Metering does not manage or delete Amazon S3 bucket data. After uninstalling metering, you must manually clean up S3 buckets that were used to store metering data.
 ====
 
 [id="metering-remove"]
 == Removing the Metering Operator from your cluster
 
-Remove the Metering Operator from your cluster by following the documentation on
-xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
+Remove the Metering Operator from your cluster by following the documentation on xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
 
 [NOTE]
 ====
-Removing the Metering Operator from your cluster does not remove its CustomResourceDefinitions or managed resources. See the following sections on xref:../metering/metering-uninstall.adoc#metering-uninstall_metering-uninstall[Uninstalling a metering namespace]
-and xref:../metering/metering-uninstall.adoc#metering-uninstall-crds_metering-uninstall[Uninstalling metering CustomResourceDefinitions]
-for steps to remove any remaining metering components.
+Removing the Metering Operator from your cluster does not remove its custom resource definitions or managed resources. See the following sections on xref:../metering/metering-uninstall.adoc#metering-uninstall_metering-uninstall[Uninstalling a metering namespace] and xref:../metering/metering-uninstall.adoc#metering-uninstall-crds_metering-uninstall[Uninstalling metering custom resource definitions] for steps to remove any remaining metering components.
 ====
```

Comment:
Removed hard line wraps. No other updates to `metering/metering-uninstall.adoc`.